Разметка всех страниц

evgeniy-kirilchik opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2147569/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/2147569-device-35/2/

evgeniy-kirilchik committed:
Разметка хейдер

Разметка хейдер

evgeniy-kirilchik committed:
Update index.html

Разметил слайдер в мейн

evgeniy-kirilchik committed:
Update index.html

Доразметил мэйн
Разметил футер

evgeniy-kirilchik committed:
index.html

Правки в навигации

evgeniy-kirilchik committed:
Update catalog.html

Разметил один из фильтров

evgeniy-kirilchik committed:
Update catalog.html

Доразметил каталог

evgeniy-kirilchik committed:
Update catalog.html

Правки после проверки валидатором

LeMarck commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+          <li> <img class="img-navigation-logo" src="#" alt="Лого I"></li>
+          <li> <img class="img-navigation-logo" src="#" alt="Лого C"></li>
+          <li> <img class="img-navigation-logo" src="#" alt="Лого E"></li>
+        </ul>
```
Вот такое я впервые вижу)))
Это просто ссылка и картинкой внутри

LeMarck commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+  <body>
+    <header class="page-header">
+      <h1 class="visually-hidden">Каталог магазина</h1>
+      <nav class="navigation">
```
Навигация в этом макете, это не вся шапка, а только разделы "Каталог товаров", "Доставка", "Гарантии", "Контакты". Считай, что nav - это меню

LeMarck commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+          <li class="navigation-user-menu-item">
+            <form class="navigation-form-search" action="#" method="get">
+            <img src="#">
+            <p><input type="search" name="search" placeholder="Текст поиска"></p>
```
Оборачивать инпут в `p` (текстовый элемент) плохо!!! Убери эту обёртку

LeMarck commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+          <li> <img class="img-navigation-logo" src="#" alt="Лого C"></li>
+          <li> <img class="img-navigation-logo" src="#" alt="Лого E"></li>
+        </ul>
+        <ul class="navigation-list navigation-user-menu">
```
А ещё это не список, а трёхколоночник: поисе, авторизация, блок экшенов

LeMarck commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+        <ul class="navigation-list navigation-user-menu">
+          <li class="navigation-user-menu-item">
+            <form class="navigation-form-search" action="#" method="get">
+            <img src="#">
```
Напишу только тут, но касается это всех `img`. В стандарте у img есть всего один обязательный атрибут и это не `src`, а  `alt` который ты тут и дальше забыл

LeMarck commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+            <button class="button-search" type="submit">Найти</button>
+          </form>
+          </li>
+          <li class="navigation-user-menu-item"> <img src="#">
```
Ещё иконки эти не являются контентными, они декоративные и будут вставляться через CSS

LeMarck commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+        </ul>
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <p>Каталог товаров</p>
```
Это тоже ссылка и ведёт она в `catalog.html`

LeMarck commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+    </header>
+    <main class="main-catalog">
+      <section class="breadcrumbs">
+        <p><b>Моноподы для селфи</b></p>
```
Если хочется сделать текст жирным, то не надо его оборачивать в `b`, этот тэг для акцентирования

И вообще это не просто текст, а заголовок страницы – `h1`

LeMarck commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+          <li class="breadcrumbs-list-item"> <a href="catalog.html">Каталог товаров</a></li>
+          <li> <img src="#"> </li>
+          <li class="breadcrumbs-list-item"> <a href="#">Моноподы для селфи</a></li>
+        </ol>
```
**O**rdered **L**ist

В данном случае это просто `ul`(**U**nordered **L**ist) Это не нумерованный список

LeMarck commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+        <form class="catalog-filter" action="https://htmlacademy.ru/" method="get">
+           <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title"> Стоимость</legend>
+            <input type="">
```
Давай пока дадим тип `range`

LeMarck commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+          <option value="expensive">Дорогие</option>
+        </select>
+        <button class="sort-item" type="button">sort up</button>
+        <button class="sort-item" type="button">sort down</button>
```
Давай эти виды сортировки сделаем не кнопками, а ссылками

LeMarck commented on index.html:
```diff
@@ -2,11 +2,183 @@
+        </section>
+    <section class="slider-product-description">
+         <p><b>Порхает как бабочка,
+            жалит как пчела!</b></p>
```
Выглядит как заголовок второго уровня

LeMarck commented on index.html:
```diff
@@ -2,11 +2,183 @@
+          50 м Радиус поражения
+         </p>
+        </section>
+        <h2 class="visually-hidden">Список категорий товаров</h2>
```
Заголовки секций должны быть в самих секциях

LeMarck commented on index.html:
```diff
@@ -2,11 +2,183 @@
+            <p><b>Доставка</b></p>
+            <p>Мы с удовольствием доставим ваш товар прямо к вашему подъезду совершенно бесплатно! Ведь мы неплохо заработаем, поднимая его на ваш этаж.</p>
+            <img src="#">
+          </blockquote>
```
Клёво, что знаешь такой тэг, но тут он не уместен. Это не отзывы, тут нет цитирования

LeMarck commented on index.html:
```diff
@@ -2,11 +2,183 @@
+        </section>
+        <section class="delivery-block">
+          <img src="#">
+          <p><b>Доставим редкий товар под заказ</b></p>
```
Тоже нужно завернуть в заголовок, если это у тебя отдельная секция

LeMarck commented on index.html:
```diff
@@ -2,11 +2,183 @@
+        <section class="about-us">
+        <h2 class="visually-hidden">О нас</h2>
+        <img src="#">
+        <p><b>О нас</b></p>
```
Вот в чём прикол сначала создавать скрытый заголовок, а потом писать вот такую дичь в виде параграфа с b внутри
Удали лишнее, покажи заголовок

Контакты аналогично

LeMarck commented on index.html:
```diff
@@ -2,11 +2,183 @@
+        </section>
+        <section class="about-us">
+        <h2 class="visually-hidden">О нас</h2>
+        <img src="#">
```
Я так понимаю, что тут должна быть чёрточка

Убери этот тэг. Всё что можно нарисовать рисуем через CSS. Грузить с сервера всякие мелкие детали которые легко делаются вёрсткой и стилями – лишняя трата ресурсов

LeMarck commented on index.html:
```diff
@@ -2,11 +2,183 @@
+          Заодно сможете проверить работоспособность покупки.</p>
+        <p><b>Интернет-магазин «Девайс»</b></p>
+        <p>Санкт-Петербург, набережная реки Карповки, 5, литера П.</p>
+        <p>+7 (812) 812-12-12</p>
```
Телефон тоже сделай ссылкой `tel:`, так человек сможет тыкнув на него сразу позвонить

LeMarck commented on index.html:
```diff
@@ -2,11 +2,183 @@
+        <p><b>Контакты</b></p>
+        <p>Вы можете забрать товар сами, заехав в наш офис.
+          Заодно сможете проверить работоспособность покупки.</p>
+        <p><b>Интернет-магазин «Девайс»</b></p>
```
Подзаголовок `h4`

LeMarck commented on index.html:
```diff
@@ -2,11 +2,183 @@
+      </section>
+      <section class="subscribe-newsletter">
+        <h2 class="visually-hidden">Подписаться на рассылку</h2>
+        <p><b>Подпишитесь на рассылку </b></p>
```
Я вот ни как не могу понять откуда вот таке берётся? Вам на лайве такое точно не показывали. Зачем скрывать заголовок, а потом его дублировать, но оборачивать в параграф с акцентированием?

LeMarck commented on index.html:
```diff
@@ -2,11 +2,183 @@
+          </ul>
+        </section>
+        <section class="tel-logo-html">
+          <p>Тел.: +7 (812) 812-12-12</p>
```
Ссылка

LeMarck reviewed (commented):
Отформатируй пожалуйста код. В VS Code это делается через `Option + Shift + F` для мака и `Shift + Alt + F` для винды

evgeniy-kirilchik committed:
Update index.html

Вставил ссылку на телефон

evgeniy-kirilchik committed:
Update catalog.html

Вставил ссылку на телефон

evgeniy-kirilchik committed:
Update index.html

Исправил атрибут всех img, в футере к лого HTML Academy добавил ссылку кнопку

evgeniy-kirilchik committed:
Update catalog.html

Исправил атрибут всех img, в футере к лого HTML Academy добавил ссылку кнопку

evgeniy-kirilchik committed:
Update catalog.html

Исправил ol на ul,доправил класс иконки "плюс", убрал обертку P, изменил разметку Лого.

evgeniy-kirilchik committed:
Update index.html

добавил значение alt в лого academy  в футере, доправил класс иконки "плюс",убрал обертку P, изменил разметку лого в хедере

evgeniy-kirilchik committed:
Update catalog.html

добавил значение alt в лого academy  в футере

evgeniy-kirilchik committed:
Update index.html

Каталог товаров обернул в ссылку, обозначил заголовок слайдера, убрал теги hr и blockquote, обернул заголовок "Доставим редкий товар под заказ" в ссылку(кнопку),убрал лишние невидимые заголовки, добавил подзаголовок h3, переместил заголовки секций в них самих, изменил разметку лого в футере.

evgeniy-kirilchik committed:
Update catalog.html

Каталог товаров обернул в ссылку, добавил заголовок h1, добавил фильтру тип range,обернул сортировочные элементы в ссылку, изменил разметку лого в футере.

evgeniy-kirilchik committed:
Update catalog.html

Доправил форму поиска

evgeniy-kirilchik committed:
Update index.html

Доправил user menu

evgeniy-kirilchik committed:
Update catalog.html

Доправил user menu

evgeniy-kirilchik committed:
Update index.html

правки

evgeniy-kirilchik committed:
Update catalog.html

правки

LeMarck commented on index.html:
```diff
@@ -2,11 +2,145 @@
+          <button class="button-services-item" type="button">Доставка</button>
+          <button class="button-services-item" type="button">Гарантия</button>
+          <button class="button-services-item" type="button">Кредит</button>
+          <p><b>Доставка</b></p>
```
Это можно сделать заголовком 3 уровня

LeMarck commented on index.html:
```diff
@@ -2,11 +2,145 @@
+        <h3>Интернет-магазин «Девайс»</h3>
+        <p>Санкт-Петербург, набережная реки Карповки, 5, литера П.</p>
+        <a href="tel:78128121212">Тел.: +7 (812) 812-12-12</a>
+        <p><b>Время работы</b></p>
```
Тоже h3
